pytest_logger/plugin.py

```diff
@@ -14,6 +15,16 @@
     string_type = str
 
 
+@pytest.hookimpl
+def pytest_addoption(parser):
+    """Add options to control logger"""
+    parser.addini(
+        name='logger_logdir',
+        help='base directory with log files for file loggers [basetemp]',
+        default=None,
+    )
+    parser.getgroup('logger').addoption('--logger-logdir', dest='logger_logdir')
```

Comment:
Consider changing "logdir" in ini/cmdline option name to "logsdir".
Logdir resembles fixture name which returns path to test-specific dir.
Logsdir resembles name of test session tmpdir's subdir (named "logs") and a couple of identifiers in this module (e.g. self._logsdir or _make_logsdir*).